### PR TITLE
TST fix CI tests after rebase on main

### PR DESCRIPTION
**Pull request by beckermr**
This PR fixes the CI tests after the galsim branch was rebased on main.

### CODE REVIEW

File: jax_galsim/core/utils.py [hunk 1]
```diff
@@ -51,3 +63,42 @@ def ensure_hashable(v):
...
+            low,
+            flow,
+            mid,
+            fmid,
+            high,
+            fhigh,
+        )
+
+    low = 0.0
+    high = 1e5
```
Comment (beckermr) [inline jax_galsim/core/utils.py:100]:
```suggestion
```

**Comment by beckermr** [comment]
I'm working the merge with main here.

File: jax_galsim/transform.py [hunk 2]
```diff
@@ -105,6 +105,10 @@ def flux_ratio(self):
     def _flux(self):
         return self._flux_scaling * self._original.flux
 
+    @property
+    def _offset(self):
+        return self._params["offset"]
+
```
Comment (beckermr) [inline jax_galsim/transform.py:111]:
```suggestion
```

**Comment by beckermr** [comment]
OK @ismael-mendoza this one is ready for review!

**Comment by beckermr** [comment]
I am not sure. How much longer? A few minutes?

**Comment by beckermr** [comment]
Looks like ~2x! I suspect it is the changes to the Moffat class to ensure we match the new maxk computations from galsim. I'll look later.

**Comment by beckermr** [comment]
OK I went through a bunch of old test suites. As far as I can tell, there is no big regression here. As we've added features, more of the galsim test suite is running and so things go longer. Here are some line items

 - jax_galsim.random - the tests for these classes added ~2 minutes to the test suite
 - jax_galsim.Deconvolve - once the tests for this were working, they added ~1 minute to the test suite
 - jax_galsim.Image.wrap - the tests for this method added ~1 minute to the test suite
 - tests/jax/test_api.py - these tests added ~1 minute to the test suite

Those alone accoutn for ~half the time.

**Comment by beckermr** [comment]
This one is ready to merge @ismael-mendoza!

**Review by ismael-mendoza — APPROVED** [review]
LGTM!

**Pull request merged**